core/src/lib/internal/CoreStitchAppClient.ts

```diff
@@ -15,19 +15,19 @@ export default class CoreStitchAppClient {
     this.routes = routes;
   }
 
-  public callFunctionInternal<T>(name: string, ...args: any[]): Promise<T> {
+  public callFunctionInternal<T>(name: string, args: any[]): Promise<T> {
     return this.authRequestClient.doAuthenticatedJSONRequest(
       this.getCallFunctionRequest(name, args)
     );
   }
 
   private getCallFunctionRequest(
     name: string,
-    ...args: any[]
+    args: any[]
```

Comment:
Got rid of variadic/rest parameters here, since it was triply nesting the variadic args into something that looked like `[[["the", "actual", "arguments"]]]`. The interface in the web module is still exposed as variadic/rest parameters, but I'm open to making them non-variadic for the sake of consistency with the other SDKs. Would like your thoughts.